thirdeye/thirdeye-frontend/app/pods/components/detection-health/template.hbs

```diff
@@ -1,84 +1,103 @@
-<ul class="te-horizontal-cards__single-card">
-  <li class="te-horizontal-cards__card-title">
-    <label for="select-dimension" class="control-label te-label--no-bottom">
-      30-day status
+<div class="te-horizontal-cards te-content-block__detection-health">
+  <h4 class="te-self-serve__block-title">
+    <label for="select-dimension" class="control-label te-label">
+      Detection health
       <span>
         <i class="glyphicon glyphicon-question-sign"></i>
-        {{#tooltip-on-element class="te-tooltip"}}
-          The overall health status of your detection configuration is based on
-          the most recent 30 days.
-        {{/tooltip-on-element}}
+          {{#tooltip-on-element class="te-tooltip"}}
+            See how your detection configuration is performing in detail
+            by clicking 'view details' below{{#if showRules}},
+            and select different rules above the graph
+            to see their respective regression errors{{/if}}.
+          {{/tooltip-on-element}}
       </span>
     </label>
-  </li>
-  <li class="te-horizontal-cards__card-number{{overall.label}}">
-    {{overall.status}}
-  </li>
-  <a role="button" tabindex="0" class=".te-button--link">
-    view details
-    {{#bs-popover placement="bottom" title="Detection health"}}
-      <p>Overall health of the metrics that monitor detection performance.</p>
+  </h4>
+  <div class="te-horizontal-cards__container">
+    <ul class="te-horizontal-cards__single-card">
+      <li class="te-horizontal-cards__card-title">
+        <label for="select-dimension" class="control-label te-label--no-bottom">
+          30-day status
+          <span>
+            <i class="glyphicon glyphicon-question-sign"></i>
+            {{#tooltip-on-element class="te-tooltip"}}
+              The overall health status of your detection configuration is based on
+              the most recent 30 days.
+            {{/tooltip-on-element}}
+          </span>
+        </label>
+      </li>
+      <li class="te-horizontal-cards__card-number{{overall.label}}">
```

Comment:
do you need a hyphen here to meet the BEM requirement?

Reply:
No, the `overall.label` points to the modifier like `--good` `--bad` which is defined in js.